invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/api/DepositApiClient.js

```diff
@@ -322,10 +348,27 @@ export class DepositFileApiClient {
  * Default File API Client for deposits.
  */
 export class RDMDepositFileApiClient extends DepositFileApiClient {
-  initializeFileUpload(initializeUploadUrl, filename) {
+  constructor(additionalApiConfig, defaultTransferType, transferTypes) {
+    super(additionalApiConfig);
+    this.defaultTransferType = defaultTransferType || "L";
+    this.transferTypes = transferTypes || ["L"];
+  }
+
+  initializeFileUpload(initializeUploadUrl, filename, transferOptions = {}) {
+    const { fileSize = 0, type = this.defaultTransferType, ...opts } = transferOptions;
+
+    if (!this.transferTypes.includes(type)) {
+      throw new UnsupportedTransferTypeError(
```

Comment:
I think we lost the comments from the other PR, so I will try to recreate it.
could we remove the possibility of multi[art upload if it is not configured on the backend? you can pass the config via jinja context variable to achieve this

Reply:
is the exception still needed then ? could it be removed if not used?